common/query_assist/index.ts

```diff
@@ -0,0 +1,18 @@
+import { TimeRange } from '../../../../src/plugins/data/common';
+
+export const ERROR_DETAILS = { GUARDRAILS_TRIGGERED: 'guardrails triggered' };
+
+export const SUPPORTED_LANGUAGES = ['PPL'] as const;
```

Comment:
good idea. i'll modify the schema a bit since query assist also needs the agent config name
https://github.com/kavilla/queryEnhancements/blob/9b811a5459b63a3c8fc29ee25003df47df68c5e6/server/routes/query_assist/index.ts#L6